metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -61,6 +61,8 @@ EOF
 function remote_network_bandwidth_iperf3 {
 	test_name="Remote network bandwidth"
 	get_runtime
+	init_env
```

Comment:
Unless it happens from a generic lib function, can you consider adding the `check_images` calls as well as added in #773